common/schema-model/src/main/java/io/smallrye/graphql/schema/model/Schema.java

```diff
@@ -96,51 +97,57 @@ public boolean hasSubscriptions() {
     }
 
     public Map<Group, Set<Operation>> getGroupedQueries() {
-        return groupedQueries;
+        Set<Operation> operations = namespaceQueries.values().stream()
+                .map(GroupContainer::getAllOperations)
+                .flatMap(Collection::stream)
+                .collect(Collectors.toSet());
+        return Map.of(new Group(), operations);
```

Comment:
I understand that you moved the nesting logic into the `GroupContainer` class, but I don't really get, why we then still need a `Map` with an "empty" `Group`... wouldn't it be enough to somehow work with a `GroupContainer`? Maybe this could simplify a lot of things in here.

Reply:
just for quarkus

Reply:
I love Quarkus, but making this code simpler also makes the Quarkus code simpler, wouldn't it?

Reply:
I returned this part as it was (for Group). 
As for Map<String, NamesapceContainer> - I will try change it for Set